packages/cli/test/mocks/project.ts

```diff
@@ -1,8 +1,7 @@
 import { client } from './client';
-import {
-  Project,
+import type {
   ProjectEnvTarget,
-  ProjectEnvType,
+  Project,
```

Comment:
**nitpick (non-blocking):** Sort imports alphabetically?

Reply:
If we want to enforce that we need to enable it using eslint or something.